### DISCUSSION TITLE
Load model but still keep poster

### DISCUSSION THREAD

**Discussion by vydimitrov** [google/model-viewer#4766] [General]
Hello all, thanks for the great package. I really enjoy playing with it and the 3D models.

I have a question since I wasn't able to find an answer on the internet. Is it possible to load the model and still keep the poster visible? The use case I have, I want to load the model, hide some parts/materials of it and then remove the poster. The poster has the same image as the model without the parts. 

The problem I have currently is that once the model is loaded the poster is removed, and for a blink of a eye I get to see the parts I want to remove. If I set the reveal to manual the model is never loaded.  

Thank you,

**Comment by elalish** [comment]
I believe the combination you're looking for is `reveal="manual"` and `loading="eager"`.

**Reply by vydimitrov** [reply]
Wow great, that was it! Thank you. I thought the `loading` prop had a similar effect as the image loading.
